<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchprescriptionFunc</class>
 <widget class="QWidget" name="SearchprescriptionFunc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="PatientId">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>PatientId</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SearchBtn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>查询处方</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="prescriptionInfo">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>230</y>
     <width>104</width>
     <height>70</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="closeBtn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭界面</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
